backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/containers.js

```diff
@@ -40,6 +40,21 @@ Containers.canDeleteContainer = async (req, res, next) => {
 	}
 };
 
+Containers.checkModelStatus = async (req, res, next) => {
+	try {
+		const { teamspace, container } = req.params;
+		const { status } = await getModelById(teamspace, container, { _id: 0, status: 1 });
+
+		if (status === 'queued' || status === 'processing') {
```

Comment:
do it the other way round, we have more status messages than queued and processing.

So it's safer (and more expandable to say) if status is not `ok`, `undefined` or `failed`

Reply:
Can we add model status as a constant in `modelSettings.constants` and use it from there?